test/Libraries/GeometryColorTests/GeometryImportHelperTests.cs

```diff
@@ -31,6 +31,7 @@ protected override void GetLibrariesToPreload(List<string> libraries)
         }
 
         [Test]
+        [Category("Failure")]
         public void SATAndSABUnitImportNodeModelNodesWork_PartialApplication_Replication()
```

Comment:
Any idea what's the failure?

Reply:
The value coming back is zero. See logs referenced in the PR below.